AGM_Interaction/config.cpp

```diff
@@ -186,6 +187,7 @@ class AGM_Core_Options {
 
 class AGM_Parameters {
   AGM_Modifier = 0;
+  AGM_enableTeamManagement = 1;
```

Comment:
We usually prefix parameters with the module name: `AGM_Interaction_enableTeamManagement`